file/delete_scheduler.cc

```diff
@@ -51,6 +51,9 @@ DeleteScheduler::~DeleteScheduler() {
   if (bg_thread_) {
     bg_thread_->join();
   }
+  for (const auto& it : bg_errors_) {
+    it.second.PermitUncheckedError();
```

Comment:
I am just wondering about this. Doing it in the destructor makes me think that it is not important if a background error occurs or or not? Is that the case?

Reply:
I cannot find any code that actually checks these errors outside of the tests, so I am thinking that background errors might not be that important.  But this is new code to me, so I might be missing something...